components/notability/commons/notability_checker.lua

```diff
@@ -102,7 +102,8 @@ end
 
 function NotabilityChecker._calculatePlayerNotability(player)
 	player = mw.ext.TeamLiquidIntegration.resolve_redirect(player)
-	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]'
+	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]' ..
+		' OR [[participant::' .. player .. ']]'
 	for i = 2, 10 do
```

Comment:
the `10` here would need to be adjustable for several wikis
some might have >10 players listed in a teamCard (and all of them playing in some of the matches of the event)

Reply:
We'll make it 20 🤷‍♀️

Reply:
atm sc2 allows 30 (but as we atm store it differently in lpdb it is not as relevant for us)